### PR TITLE
qf-reloader-dev: add reloader/fs_sync to dev env only

### PR DESCRIPTION
**Pull request by icehess**

### CODE REVIEW

File: make/deps.mk [hunk 1]
```diff
@@ -41,7 +41,7 @@ dep_hep = git https://github.com/lazedo/hep 1.5.4
 dep_ecsv = git https://github.com/lazedo/ecsv ecsv-1
 
 ### for scripts/dev-start-*.sh
-## dep_reloader = git https://github.com/oinksoft/reloader
+dep_reloader = git https://github.com/oinksoft/reloader
```
Comment (lazedo) [inline make/deps.mk]: get a tag

Comment (icehess) [inline make/deps.mk]: there is no tag

File: make/deps.mk [hunk 2]
```diff
@@ -27,11 +27,11 @@ dep_couchbeam = git https://github.com/lazedo/couchbeam 1.3.1.2
 ## other to come
 ## there are some commits after 1.4.0 that will go into 1.5.0
 ## that break a loot of stuff. please be careful if updating to 1.5.0
-## from for-GET/jesse 
+## from for-GET/jesse
 dep_jesse = git https://github.com/lazedo/jesse 1.4.1
 
 dep_lager = git https://github.com/basho/lager 3.2.1
-## dep_fs_sync = git https://github.com/jamhed/fs_sync
+dep_fs_sync = git https://github.com/jamhed/fs_sync
```
Comment (lazedo) [inline make/deps.mk]: get a commit

File: make/deps.mk (hunk 1 above)
Comment (lazedo) [inline make/deps.mk]: get a commit

**Comment by lazedo** [comment]
since fs_sync gets fs_event, please also get a commit for fs_event

**Comment by icehess** [comment]
done

**Pull request merged**